frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -25,22 +25,23 @@
                             :error="item.error"
                             :disabled="item.encrypted"
                             value-empty-text=""
-                            :type="(!readOnly && (editTemplate || item.policy === undefined))?'text':'uneditable'"></FormRow>
-                        <!-- <ff-text-input  v-model="item.name" :disabled="item.encrypted"  /> -->
+                            :type="(!readOnly && (editTemplate || item.policy === undefined)) ? 'text' : 'uneditable'"
+                        ></FormRow>
                     </td>
-                    <td class="px-4 py-4 border w-auto align-top" :class="{'align-middle':item.encrypted}">
-                        <div class="w-full" v-if="!item.encrypted">
+                    <td class="px-4 py-4 border w-auto align-top" :class="{'align-middle': item.encrypted}">
+                        <div class="w-full" v-if="settings['team:environment-variable-view'] || user.admin">
```

Comment:
I believe we'll still want the `item.encrypted` feature to show the padlock icon. The `settings['team:environment-variable-view']` can be added as an extra fork the if statement

Reply:
@Pezmc  i have tried  ```v-if="!item.encrypted"``` but its not hiding variable values, if i removed not(!) then its works but padlock icon is not visible .

Reply:
![WhatsApp Image 2023-06-28 at 8 27 21 PM](https://github.com/flowforge/flowforge/assets/110285294/da4a4144-729a-4367-a050-5d58ca95d176)

done now pedlock icon is visible